packages/chatbot/src/host/exchanger.ts

```
@@ -0,0 +1,326 @@
+    const sendDelay = sendBackoffInterval(store.stats.sendErrors)
+    log.debug('Error detected. Send delay increased to {} ms', sendDelay)
+
+    store.isSending = false
+
+    if (store.isEnabled)
+      startTicker(sendTimerName, sendDelay)
+
+  }
+
+  // === Правило: Отправка исходящих сообщений ===
+  defineRule(sendTimerName, {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)